Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.h

```diff
@@ -298,6 +298,7 @@ class WebChromeClient final : public WebCore::ChromeClient {
 
 #if PLATFORM(MAC)
     std::unique_ptr<WebCore::ScrollbarsController> createScrollbarsController(WebCore::Page&, WebCore::ScrollableArea&) const final;
```

Comment:
Would be nice to avoid both `create` and `update` functions. Maybe just an `ensure` function that does both?

Reply:
Fixed.